### DISCUSSION TITLE
Allow windows to maximize behind notch on newer MacBook Pros

### DISCUSSION THREAD

**Discussion by Tobbe** [rxhanson/Rectangle#1290] [Ideas]
There are a couple of issues about this already (https://github.com/rxhanson/Rectangle/issues/695, https://github.com/rxhanson/Rectangle/issues/827), but they've both been closed. And I think something has changed in MacOS, so I'm opening up a discussion about it.

Previous issues said it wasn't possible to place windows behind the notch. This is not the case now. By manually resizing a window I can make it be proper full screen. See photo below.

![PXL_20231114_152014027](https://github.com/rxhanson/Rectangle/assets/30793/9cd43783-7c9b-408f-9071-445acbb6617a)

So maybe Rectangle could now just simply resize windows like as if they were manually resized?

If that doesn't work, I also found this blog post with a bunch of technical jargon I didn't understand 😅  But hopefully you do @rxhanson 😄 
https://notes.alinpanaitiu.com/Fullscreen-apps-above-the-MacBook-notch

**Comment by rxhanson** [comment]
Thanks for the feedback. 

Although you can resize the window under the notch, the Accessibility API that Rectangle uses to place windows doesn't support that, unless there is a workaround that I'm unaware of. 

I still don't have plans to add this in myself, but don't mind merging a good PR for it if someone has a good way to do it.

That post by Alin demonstrates moving the displayed portion of the screen below the notch (probably not what you're looking for?)

**Reply by Tobbe** [reply]
I want what Alin shows in this video

https://files.alinpanaitiu.com/parallels-full-notch-h264.mp4
![image](https://github.com/rxhanson/Rectangle/assets/30793/81b82ec6-b70d-4790-9234-b565920e6f37)

And this one

https://files.alinpanaitiu.com/noteplan-full-notch-h264.mp4
![image](https://github.com/rxhanson/Rectangle/assets/30793/2cb6ae74-35be-4096-a757-3749df364dc0)


Where the app truly fits the whole screen (and the notch covers parts of the app's UI)

But yeah, requiring disabling SIP isn't going to fly 🙁

**Reply by Tobbe** [reply]
> I still don't have plans to add this in myself

Understood 👍 
Thanks for taking the time to reply to my message! 🙂
